sql/S1_CREATE_DOMAINS.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DOMAIN user_name AS
+  VARCHAR(32) NOT NULL CHECK (VALUE ~ '^[[:alpha:]]+(([''.\s-][[:alpha:]\s])?[[:alpha:]]*)*$');
```

Comment:
What is the purpose of `~`?

Reply:
it is a regex operator that returns true if regex match, false otherwise